# ObjectDetections.msg
# 多个物体的检测结果列表

std_msgs/Header header
ObjectDetection[] objects  # 所有检测到的物体列表

